src/import/customer-import.service.ts

```diff
@@ -82,6 +84,30 @@ export class CustomerImportService {
       });
     }
 
+    for await (const ordersBatch of this.orderImportService.getAllOrders(
+      period,
+    )) {
+      ordersBatch.forEach((order) => {
+        if (order.paymentState !== 'Paid') {
+          return;
+        }
+
+        customers.push({
+          object: 'customer',
+          source_id: order.customerId || order.anonymousId,
```

Comment:
Will we overwrite customer details with contact details if an order is connected with the customer?